examples/Makefile

```diff
@@ -401,7 +401,7 @@ ifeq ($(TARGET_PLATFORM),PLATFORM_DESKTOP_GLFW)
     ifeq ($(PLATFORM_OS),BSD)
         # Libraries for FreeBSD, OpenBSD, NetBSD, DragonFly desktop compiling
         # NOTE: Required packages: mesa-libs
-        LDLIBS = -lraylib -lGL -lpthread -lm
+        LDLIBS = -L /usr/pkg/lib -L /usr/X11R7/lib -Wl,-R/usr/pkg/lib -Wl,-R/usr/X11R7/lib -lraylib -lGL -lpthread -lm
```

Comment:
Library paths (`-L`) should be added to `LDPATHS` and compiler checks should be added to `CFLAGS`, `LDLIBS` is only intended for libraries (`-l`).

Reply:
should be fixed now